index.html

```diff
@@ -31,90 +33,98 @@
       </li>
-      <li>
-        <h3>красивая дорога</h3>
-        <p>— №4 —</p>
-        <p>ехать в седону из лас-вегаса совсем  не скучно!</p>
+      <li class="features-item">
+        <h3 class="features-title">красивая дорога</h3>
+        <p class="features-text">— №4 —</p>
+        <p class="features-text">ехать в седону из лас-вегаса совсем  не скучно!</p>
       </li>
-      <li>
-        <h3>мало туристов</h3>
-        <p>— №5 —</p>
-        <p>большинство едет в гранд каньон и толпится там</p>
+      <li class="features-item">
+        <h3 class="features-title">мало туристов</h3>
+        <p class="features-text">— №5 —</p>
+        <p class="features-text">большинство едет в гранд каньон и толпится там</p>
       </li>
     </ul>
   </section>
-  <section class="form">
-    <div class="form__title">
-      <h2>Заинтересовались?</h2>
-      <p class="">Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в седоне</p>
+  <section class="main-form">
+    <div class="main-form-top">
+      <h2 class="main-form-title">Заинтересовались?</h2>
+      <p class="main-form-desc">Укажите предполагаемые даты поездки,<br> и мы покажем вам лучшие предложения гостиниц в седоне</p>
     </div>
-    <div class="form__bottom">
-      <button type="button">ПОИСК ГОСТИНИЦЫ В СЕДОНЕ</button>
+    <div class="main-form-bottom">
+      <button type="button" class="main-form-button">ПОИСК ГОСТИНИЦЫ В СЕДОНЕ</button>
     </div>
   </section>
-  <section class="map">
+  <section class="main-map">
     <iframe src="" frameborder="0"></iframe>
   </section>
 </main>
-<footer>
-  <div>
-    <a href="">#visitSEDONA</a>
-    ТЕЛ: <a href="tel:+74956660166">+7(495) 666-01-66</a>
+<footer class="main-footer container">
+  <div class="main-footer-contact">
+    <a href="/" class='main-footer-link'>#visitSEDONA</a>
+    ТЕЛ: <a href="tel:+74956660166" class='main-footer-phone'>+7(495) 666-01-66</a>
   </div>
-  <ul>
+  <ul class="main-footer-social">
    <li>
     <a href="">
-      <img src="img/twitter.svg" alt="twitter" width="46" height="48">
+      <img src="img/twitter.svg" alt="twitter" width="17" height="17">
     </a>
   </li>
   <li>
     <a href="">
-      <img src="img/fb-icon.svg" alt="facebook" width="46" height="48">
+      <img src="img/fb-icon.svg" alt="facebook" width="12" height="22">
     </a>
   </li>
   <li>
     <a href="">
-      <img src="img/youtube.svg" alt="youtube" width="46" height="48">
+      <img src="img/youtube.svg" alt="youtube" width="20" height="16">
     </a>
   </li>
 </ul>
-<a href="https://htmlacademy.ru/intensive/htmlcss" target="_blank">
-  <img src="img/logo-htmlacademy.svg" alt="htmlacademy" width="115" height="41">
+<a href="https://htmlacademy.ru/intensive/htmlcss" target="_blank" class="main-footer-logo">
+  <span class="main-footer-logo-text">Website by</span>
+  <svg width="115" height="41" viewBox="0 0 115 41" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
лишний атрибут в разметке
```suggestion
  <svg width="115" height="41" viewBox="0 0 115 41" fill="none">
```